x-pack/plugins/observability/public/components/shared/date_picker/index.tsx

```diff
@@ -96,7 +95,9 @@ export function DatePicker({ rangeFrom, rangeTo, refreshPaused, refreshInterval
       refreshInterval={refreshInterval}
       onRefreshChange={onRefreshChange}
       commonlyUsedRanges={commonlyUsedRanges}
-      onRefresh={onTimeChange}
+      onRefresh={() => {
+        onRefreshTimeRange();
+      }}
```

Comment:
tiny nit (just to keep in consistent with the other handlers)
```suggestion
      onRefresh={onRefreshTimeRange}
```